src/applications/facility-locator/types/index.js

```diff
@@ -0,0 +1,184 @@
+import PropTypes from 'prop-types';
```

Comment:
We had so many lint warnings about missing prop types. So I pseudo-Typescript'd the types.